content/docs/getting-started@ja.md

```diff
@@ -0,0 +1,96 @@
+---
+$title: はじめに
+$order: 0
+toc: true
+---
+[TOC]
+
+これは、 AMP を始めるためのクイックスタートガイドです。
+
+詳細な説明については [初めての AMP ページを作成する](/docs/tutorials/create.html) チュートリアルをご覧ください。
```

Comment:
Point to the Japanese tutorial by prepending link with "/ja", like `/ja/docs/tutorials/create.html`